test/MusicStore.Test/ManageControllerTest.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
I vaguely remember license header is not required for samples. @Eilon is the license header required for the samples?